File: Python/tests/test_oms_par4.py
import pytest

from hbayesdm.models import oms_par4


def test_oms_par4():
    _ = oms_par4(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_oms_par4.R
context("Test oms_par4")
library(hBayesDM)

test_that("Test oms_par4", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(oms_par4(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
